jass/systems/trade/TradeShipSupervisor.zn

```diff
@@ -187,22 +252,42 @@ library TradeShipSupervisor requires IDUtils, TradeShipSpawnOrder, TradeShipStop
             if (GetRandomReal(0, 1) < 0.5) {
                 nw = top;
                 nwPrev = left;
-                se = right;
-                sePrev = bottom;
+                se = bottom;
+                sePrev = right;
             } else {
                 nw = left;
                 nwPrev = top;
-                se = bottom;
-                sePrev = right;
+                se = right;
+                sePrev = bottom;
             }
 
             nextShip = GetTradeShipSpawnOrder();
 
             Map.onGameStart(function() {
-                if (AllowTrade) {
+                if (GameConfig.getInstance().getTradeEnabled()) {
                     TradeShipSupervisor.spawn();
                 }
             });
+
+            ChatCommands.registerArgFunc(null, "trade-routes", function (ArgsList a) {
+                DisplayTimedTextToPlayer(a.triggerPlayer, 0, 0, 10, GENERAL_COLOR + "At the start of the game, two boats will spawn from opposite sides of the map");
+                DisplayTimedTextToPlayer(a.triggerPlayer, 0, 0, 10, GENERAL_COLOR + "The first boats will randomly spawn at either N & S or E & W");

```

Comment:
Did this actually change? I think the N & E versus S & W was more balanced.

Reply:
Problem was the double boat spots though